src/Extension/StatusRuntime.php

```diff
@@ -35,9 +36,31 @@ public function addStatusService(StatusClassRendererInterface $renderer): void
     }
 
     /**
-     * @param mixed $statusType
+     * @param object|string $object     Object for StatusClassRenderer or string for FlashManager
+     * @param string|null   $statusType Object status type or Sonata flash message type
+     * @param string        $default    Default status class
+     * @param mixed         $statusType
      */
-    public function statusClass(object $object, $statusType = null, string $default = ''): string
+    public function statusClass($object, $statusType = null, string $default = ''): string
+    {
+        if (\is_object($object)) {
+            return $this->statusClassForStatusClassRenderer($object, $statusType, $default);
+        }
+
+        if (\is_string($object)) {
+            return $this->statusClassForFlashManager($object, $statusType, $default);
+        }
+
+        @trigger_error(sprintf(
```

Comment:
INHO you can throw an exception, because you allow other objects than strings

Reply:
Technicly yes, but I want keep BC with 0.x version and revert `object` type hint in 2.0, after seperate FlashManager from StatusRenderer object.